### ISSUE TITLE
[Bug]: pymilvu 2.2.2 sdk, use collection.insert(data) method,  entity_helper.py raise ParamError

### ISSUE THREAD

**Issue by bjgyl** [milvus-io/milvus#22409]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.2.3
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):default    
- SDK version(e.g. pymilvus v2.0.0rc2):pymilvus v2.2.2
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

when i use collection.insert(data) method to insert data, cause param check error:
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\entity_helper.py", line 27, in check_str_arr
    raise ParamError(message=f"invalid input, length of string exceeds max length. length: {len(s)}, max length: {max_len}")
pymilvus.exceptions.ParamError: <ParamError: (code=1, message=invalid input, length of string exceeds max length. length: 2, max length: 2)>

### Expected Behavior

when data column length == max length , shoud pass ,but throw exception,
I check the source code ,found as follow:
def check_str_arr(str_arr, max_len):
    for s in str_arr:
        if not isinstance(s, str):
            raise ParamError(message=f"expect string input, got: {type(s)}")
        if len(s) >= max_len:
            raise ParamError(message=f"invalid input, length of string exceeds max length. length: {len(s)}, max length: {max_len}")

I think "len(s) >= max_len" is wrong , the correct should be  "len(s) > max_len"

### Steps To Reproduce

```markdown
no
```

### Milvus Log

Traceback (most recent call last):
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\decorators.py", line 109, in handler
    raise e
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\decorators.py", line 105, in handler
    return func(*args, **kwargs)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\decorators.py", line 136, in handler
    ret = func(self, *args, **kwargs)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\decorators.py", line 85, in handler
    raise e
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\decorators.py", line 50, in handler
    return func(self, *args, **kwargs)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\grpc_handler.py", line 388, in batch_insert
    raise err
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\grpc_handler.py", line 370, in batch_insert
    request = self._prepare_batch_insert_request(collection_name, entities, partition_name, timeout, **kwargs)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\grpc_handler.py", line 360, in _prepare_batch_insert_request
    else Prepare.batch_insert_param(collection_name, entities, partition_name, fields_info)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\prepare.py", line 316, in batch_insert_param
    field_data = entity_helper.entity_to_field_data(entity, fields_info[location[entity.get("name")]])
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\entity_helper.py", line 73, in entity_to_field_data
    field_data.scalars.string_data.data.extend(entity_to_str_arr(entity, field_info, True))
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\entity_helper.py", line 39, in entity_to_str_arr
    check_str_arr(arr, max_len)
  File "D:\ProgramFiles\anaconda3\lib\site-packages\pymilvus\client\entity_helper.py", line 27, in check_str_arr
    raise ParamError(message=f"invalid input, length of string exceeds max length. length: {len(s)}, max length: {max_len}")
pymilvus.exceptions.ParamError: <ParamError: (code=1, message=invalid input, length of string exceeds max length. length: 2, max length: 2)>

### Anything else?

no

**Comment by XuanYang-cn** [comment]
@bjgyl It's already fixed on pymilvus master, will be available at pymilvus 2.3.0 and pymilvus 2.2.3
/unassign
/assign @bjgyl

**Comment by alankashkash** [comment]
Hey, @XuanYang-cn when do you plan on relasing pymilvus 2.2.3? Facing the same issue here
`pymilvus.exceptions.ParamError: <ParamError: (code=1, message=invalid input, length of string exceeds max length. length: 2094, max length: 1000)>` while my schema doesn't assume 1000 characters anywhere.
